README.md

```diff
@@ -32,6 +32,14 @@ If you want to limit the download to e.g. the last 3 episodes, do:
 
     ./sipper --user me@example.com --pw mypassword --max 3
 
+If you want to change the order of downloading from first to last, do:
+
+    ./sipper --user me@example.com --pw mypassword --asc
```

Comment:
Hm, maybe `--oldest-first` as in @zeroum's PR #15? Could be a clearer name.

Love that the PR comes with docs :)